api/base/serializers.py

```diff
@@ -68,6 +68,12 @@ def to_representation(self, obj):
             ret['self'] = obj.get_absolute_url()
         return ret
 
+
+class LinksField(LinksFieldWIthSelfLink):
```

Comment:
If anything, it seems that `LinksFieldWithSelfLink` should be a subclass of `LinksField`.